README.md

```diff
@@ -148,16 +148,16 @@ The current target GPU products with relevant instruction set architectures are:
 | gfx803        | Fiji       | Fiji XT         | AMD Radeon™ R9 Nano                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
 | gfx803        | Fiji       | Fiji XT         | AMD Radeon™ R9 Fury X                             |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
-| gfx803        | Fiji       | Fiji XT         | AMD Radeon™ Instinct MI8                          |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
-| gfx803        | Fiji       | Capsaicin XT    | AMD FirePro S9300x2                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Fiji       | Fiji XT         | AMD Radeon™ Instinct™ MI8                          |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx803        | Fiji       | Capsaicin XT    | AMD FirePro™  S9300x2                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
 | gfx803        | Fiji       | Capsaicin XT    | AMD Radeon™ PRO Duo 2016                          |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
 | gfx803        | Polaris    | Polaris 11      | AMD Radeon™ PRO WX 4100                           |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
 | gfx803        | Polaris    | Polaris 11      | AMD Radeon™ PRO WX 4130 Mobile                    |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
 | gfx803        | Polaris    | Polaris 11      | AMD Radeon™ PRO WX 4150 Mobile                    |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
 | gfx803        | Polaris    | Polaris 11      | AMD Radeon™ PRO WX 4170 Mobile                    |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
 | gfx803        | Polaris    | Polaris 11      | AMD Radeon™ RX 460                                |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
 | gfx803        | Polaris    | Polaris 11      | AMD Radeon™ RX 560D                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
-| gfx803        | Polaris    | Polaris 10      | AMD Radeon™ Instinct MI6                          |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
```

Comment:
Sorry to keep dumping weird trademark details, but when a product is "Radeon Instinct", we trademark "Radeon Instinct(tm)". When Radeon is standalone we used "Radeon(tm)", and when Instinct is alone we use "Instinct(tm)". But when used together, the term has a single tm.

Sorry, I don't make the rules, I just communicate them. :)

Reply:
Hehe, sure I won't shoot the messenger. :) I actually find that interesting: So "Radeon Instinct" is a brand of it's own in this legal sense. Cool.

Thanks for the information. I fixed this in my latest commit.